scripts/cluster/create_multinode_cluster.go

```diff
@@ -69,7 +74,7 @@ func CreateMasterKubeletService() error {
 	utils.WaitPrintf("Creating kubelet service")
 	bashCmd := `sudo sh -c 'cat <<EOF > /etc/systemd/system/kubelet.service.d/0-containerd.conf
 [Service]                                                 
-Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
+Environment="KUBELET_EXTRA_ARGS=--container-runtime=remote --v=5 --runtime-request-timeout=15m --container-runtime-endpoint=unix:///run/containerd/containerd.sock"
```

Comment:
I think we should consider hooking this `--v` to the debug option (do we have one already?). we don't want too much log activity when we run our system under high load.

Reply:
We have it only for vhive binary, used only in single node cluster. We use it for CI runners mostly.

I think we can add it to configs/setup/system.json for using it dusring setup. Please don't forget to add the description of new debug option to documentation (at least, to docs/developers_guide.md)